src/app/components/RegisterValidatorHome/components/SelectOperators/components/OperatorSelector/OperatorSelector.tsx

```diff
@@ -1,25 +1,27 @@
 import React, { useEffect, useState } from 'react';
 import { observer } from 'mobx-react';
+import { sha256 } from 'js-sha256';
+import { Grid } from '@material-ui/core';
```

Comment:
We should prefer default imports like:
`import Grid from '@material-ui/core/Grid';`
Every import per line.
The purpose of spread imports is to use them when you have something except default, like, interfaces, additional classes in one file etc.
If the purpose to import only one element it should be done through default.